brainscore_core/plugin_management/parse_plugin_changes.py

```diff
@@ -1,31 +1,40 @@
-import re
-import subprocess
-import sys
 import json
+import os
 from pathlib import Path
+import re
 from typing import List, Tuple, Dict
 
 from .test_plugins import run_args
 
 PLUGIN_DIRS = ['models', 'benchmarks', 'data', 'metrics']
+SPECIAL_PLUGIN_FILES = ['brainscore_vision/model_interface.py', 'brainscore_language/artificial_subject.py']
 
 
 def separate_plugin_files(files: List[str]) -> Tuple[List[str], List[str]]:
```

Comment:
```suggestion
def separate_plugin_files(files: List[str]) -> Tuple[List[str], List[str], List[str]]:
```
following the updated doc below